src/bindgen/ir/enumeration.rs

```diff
@@ -446,6 +467,7 @@ impl Enum {
             cfg,
             annotations,
             documentation,
+            swift_enum_macro,
```

Comment:
Given this can be computed with config+annotations, what's the point of having it in the enum struct? I might be missing something, just skimming over from the phone though

Reply:
The only point is to allow customising it with comment annotations. But I suppose it's not like people have other macros lined up ready to use instead of CF_ENUM. It's a pretty bad mount point for the syntax, such that you can't really do anything useful other than add compiler specific annotations. You can't use the macro to add a name after the enum variants, nor to add a typedef after the whole enum declaration and semicolon because enums can't be forward-declared (unlike structs/unions). As you can tell by the pretty bad macro in my other comment.

So yeah, I didn't put much thought into that, I was just copying the other Swift-related config. Will remove.